src/main/java/org/jabref/gui/fieldeditors/EditorTextField.java

```diff
@@ -19,6 +20,9 @@
 public class EditorTextField extends TextField implements Initializable, ContextMenuAddable {
 
     private final ContextMenu contextMenu = new ContextMenu();
+    private PasteActionHandler additionalPasteActionHandler = () -> {
+        // No additional paste behavior
+    };
```

Comment:
You don't need to create an interface; you can use `java.lang.Runnable` instead. Please also update in `org.jabref.gui.fieldeditors.EditorTextArea`, or remove it if it's not being used.

Reply:
i think using Runnable here would be semantically inappropriate, since it's typically used for thread-related tasks. Sticking with a more specific interface or lambda for paste handling keeps the intent clearer.

Reply:
About `Runnable`s. I also thought it's only for threading, but no.

It's a general and accepted way to have a function type that accepts to arguments and returns no type

Reply:
This is a general code style question. Does one want to have strong typing or not. - Seeing that the setting of this is very generic, I also don't see the need for an additional class type.

    () -> textField.setText(new CleanupUrlFormatter().format(new TrimWhitespaceFormatter().format(textField.getText())))

Just changing to `Runnable` should fix it?

Regarding the intention of runnable:

> Represents an operation that does not return a result.

This is excactly the thing we are looking for here, isn't it?

---

Source of Runnable:

```java
package java.lang;

/**
 * Represents an operation that does not return a result.
 *
 * <p> This is a {@linkplain java.util.function functional interface}
 * whose functional method is {@link #run()}.
 *
 * @author  Arthur van Hoff
 * @see     java.util.concurrent.Callable
 * @since   1.0
 */
@FunctionalInterface
public interface Runnable {
    /**
     * Runs this operation.
     */
    void run();
}
```

Reply:
I changed it to Runnable in my latest commit. Thanks.